releasenotes/notes/release-1.0-758fc917e40d7eb3.yaml

```
@@ -0,0 +1,81 @@
+---
+prelude: >
+  The Datadog APM Python team is happy to announce the release of v1.0.0 of
+  ddtrace.
+
+
+  The first commit to the library was made on June 20, 2016. Nearly sixty minor
+  releases later, the library has grown immensely. The library now includes over
+  60 integrations for libraries. Moreover, the library has expanded out from
+  tracing to support the Continuous Profiler, CI Visibility, and Application
+  Security.
```

```suggestion
  tracing to support the Continuous Profiler, and CI Visibility.
```

This isn't release yet, maybe we should omit?